utils/make-packages.sh

```diff
@@ -55,7 +65,10 @@ EOF
 			cat <<EOF
   * Felix ${version} (from Git commit ${sha}).
 EOF
-			git show ${version} --format=oneline -s | head -n -1 | tail -n +5 | sed 's/^/    /'
+			for tag in ${tags}; do
+			    echo "    [Changes recorded in $tag tag]"
```

Comment:
I used that format because I've seen it before in debian changelogs (for very similar reasons).  So pretty sure it's good in that context.

Less sure about the RPM spec - but there the changelog is separated off at the end and so seems less central anyway.  And in any case, there's no apparent problem in the RPM generation.

Testing-wise, we've now had a smoke pass with Debs built like this.  Don't have an RPM pass yet because I forgot to sign the packages - so let's at least wait until we get that RPM pass.